src/main/java/at/hannibal2/skyhanni/test/command/TestChatCommand.kt

```diff
@@ -0,0 +1,19 @@
+package at.hannibal2.skyhanni.test.command
+
+import at.hannibal2.skyhanni.utils.LorenzUtils
+
+object TestChatCommand {
+    fun command(args: Array<String>) {
+        if (args.isEmpty()) {
+            LorenzUtils.chat("§c[SkyHanni] Specify a chat message to test")
+            return
+        }
+        val resultList = mutableListOf<String>()
+        for (arg in args) {
+            resultList.add(arg)
+        }
+        val string = resultList.joinToString(" ")
+        LorenzUtils.chat("§a[SkyHanni] testing message: §7$string")
+        LorenzUtils.chat(string.replace("&", "§"))
```

Comment:
I would recommend you use a packet to send that message, otherwise other mods have a decent chance of not receiving that event

Reply:
Implemented :)